privacyidea/static/components/config/views/config.periodictasks.details.html

```diff
@@ -80,6 +80,20 @@
         </div>
     </div>
 
+    <div class="form-group" ng-show="ptaskid">
```

Comment:
You are hiding the checkbox, when creating a new periodic task? I.e. when ptaskid is non-existing?
Why would you do this an only allow this checkbox for existing periodic tasks?
(Maybe I am getting something wrong here?)

Reply:
Done in 969450a5